index.html

```diff
@@ -0,0 +1,86 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio: Setup and Mobile First Web App</title>
+    <link rel="stylesheet" href="css/styles.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+</head>
+<body>
+    
+    
+   
+    <div class="container">   
+        
+        <div class="ellipse-11">
+            <img src="./images/Ellipse11.png" alt="Ellipse 11">
+        </div>
+        <img src="./images/Ellipse5.png" alt="Ellipse 5" class="ellipse-5">
+        <img src="./images/Rectangle7.png" alt="Rectangle 7" class="rectangle-7">
+        <img src="/images/Rectangle4.png" alt="Rectangle 4" class="rectangle-4">
+        
+        <div class="ellipse-3">
+            <img src="./images/Ellipse3.png" alt="ellipse 3">
+        </div>
+        <img src="./images/Ellipse12.png" alt="ellipse 12" class="ellipse-12">
+        <img src="./images/Ellipse13.png" alt="Ellipse 13" class="ellipse-13">
+        <img src="./images/Ellipse4.png" alt="Ellipse 4" class="ellipse-4">
+        <img src="./images/Rectangle5.png" alt="Rectangle 5" class="rectangle-5">
+        <img src="./images/Rectangle6.png" alt="Rectangle 6" class="rectangle-6">
+        <img src="./images/Ellipse8.png" alt="Ellipse 8" class="ellipse-8">
+
+        <header class="header">
+            <h1 class="my-logo">My Logo</h1>
+            <nav class="navbar">
+                <ul>
+                    <li>
+                        <a href="#"><i class="fa fa-bars"></i></a>
+                    </li>
+                </ul>
+            </nav>
+        </header>
+    
+        <section id="#about" class="section-about">
+            <div class="welcome">
+                <h1 class="name-welcome">
+                    I’m Amare <br> Glad to see <br> you!
+                </h1>                
+            </div>
+            <div class="job-description">
+                <p class="description-of-me">
+                    I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+                </p>
+            </div>
+            
+            <div class="lets-connect">
+                <p class="lets-connect-statement">Let's connect</p>
+
+
+                <div class="social-networks">
```

Comment:
- [ ] For a list of items, always use the unordered list or `ul` tag and each list item can then be wrapped in a `li`. The list of icons thus should be wrapped in a `ul` tag instead of a `div` to maintain its semantic meaning. Kindly alter this code to use an unordered list tag instead.